experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -121,7 +121,6 @@ describe("FormAudience", () => {
     );
 
     expect(screen.getByTestId("isSticky")).toBeChecked();
-    expect(screen.getByTestId("isFirstRun")).toBeChecked();
```

Comment:
instead of removing this you can check this element is not present

Reply:
yeah so if its a desktop it should not render checkbox,

Reply:
so instead of removing line, modify it as it is not present

Reply:
tried it but showing error element not found by getByTestid ( This is because in Desktop `isfirstrun` is not present to get it so which results in this error )

Reply:
you can do something like this
    expect(testelement).not.toBeInTheDocument();

Reply:
@yashikakhurana i have made changes as you said once check it and let me know if any further changes are required